compiler/src/iree/compiler/Codegen/Common/PolynomialApproximationPass.cpp

```diff
@@ -29,6 +29,7 @@ class PolynomialApproximationPass
   void runOnOperation() override {
     RewritePatternSet mathPatterns(&getContext());
     populateExpandTanPattern(mathPatterns);
+    populateExpandExp2FPattern(mathPatterns);
```

Comment:
Could add a test case to https://github.com/openxla/iree/blob/main/tests/e2e/regression/libm_linking.mlir for this

Reply:
Done.